CommonTools/PileupAlgos/test/testPUMods.py

```diff
@@ -5,37 +5,41 @@
 process.load('FWCore/MessageService/MessageLogger_cfi')
 process.load('Configuration/StandardSequences/FrontierConditions_GlobalTag_cff')
 process.MessageLogger.cerr.FwkReport.reportEvery = 10
-from Configuration.AlCa.GlobalTag import GlobalTag
-process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_mc', '')
+process.GlobalTag.globaltag = 'START53_V7G::All'
 
 process.load('CommonTools/PileupAlgos/Puppi_cff')
 process.load('CommonTools/PileupAlgos/softKiller_cfi')
 
-process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(50) )
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(10) )
 process.source = cms.Source("PoolSource",
-                            fileNames  = cms.untracked.vstring('/store/relval/CMSSW_7_5_0_pre4/RelValQCD_FlatPt_15_3000HS_13/MINIAODSIM/MCRUN2_75_V1-v1/00000/328B19C2-4DF6-E411-92C1-003048FFD796.root')
+	fileNames  = cms.untracked.vstring(
+		# '/store/mc/RunIISpring15DR74/QCD_Pt-15to7000_TuneCUETP8M1_Flat_13TeV_pythia8/GEN-SIM-RECO/AsymptFlat0to50bx25Reco_MCRUN2_74_V9-v3/10000/0009D30B-0207-E511-B581-0026182FD753.root'
+		'file:/uscms_data/d2/ntran/physics/Jets/PUPPITuning2015/JMEValidatorWorkbench/cmssw748_pr_Aug3/src/25202.0_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25--old/step3_inMINIAODSIM.root'
```

Comment:
private files should not be in the src code. Is there a cms-wide available file?